src/mosaic.mjs

```
@@ -171,10 +171,11 @@ async function mosaic512px(z, x, y, filters = {}) {
   const metadataByUuid = {};
   await Promise.all(
     rows.map(async (row) => {
-      metadataByUuid[row.uuid] = await getGeotiffMetadata(row.uuid);
+      if (row && row.uuid) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)